src/System.Xml.XmlSerializer/ref/System.Xml.XmlSerializer.csproj

```diff
@@ -2,10 +2,10 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.10.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
Hmm, the dir.props under 'src/System.Xml.XmlSerializer' does not exist in dev/api branch yet. Will there be any FI/RI from master to dev/api so that we can use that file?